addons/project/models/project.py

```diff
@@ -533,6 +534,26 @@ def _search_is_milestone_exceeded(self, operator, value):
             operator_new = 'not inselect'
         return [('id', operator_new, (query, ()))]
 
+    @api.model
+    def _search_is_milestone_due_today(self, operator, value):
+        if not isinstance(value, bool):
+            raise ValueError(_('Invalid value: %s') % value)
+        if operator not in ['=', '!=']:
+            raise ValueError(_('Invalid operator: %s') % operator)
+
+        query = """
+            SELECT P.id
+              FROM project_project P
+         LEFT JOIN project_milestone M ON P.id = M.project_id
+             WHERE M.is_reached IS false
+               AND M.deadline = CAST(now() AS date)
```

Comment:
```suggestion
               AND M.deadline = CURRENT_DATE
```
Is equivalent, I think.